chains/avalanche/metadata.yaml

```diff
@@ -7,7 +7,7 @@ blockExplorers:
 blocks:
   confirmations: 3
   estimateBlockTime: 2
-  reorgPeriod: 3
+  reorgPeriod: 0
```

Comment:
Initially, I thought this was a typo but then checked this https://support.avax.network/en/articles/7329750-are-there-reorgs-on-avalanche

Reply:
updated back to 3, based on https://www.notion.so/hyperlanexyz/1126d35200d6804b883bcc3c95baded1?v=3043b2f883ae4e4ebba93de538351bd0